src/main/java/proj/kedabra/billsnap/business/dto/GetBillPaginationDTO.java

```diff
@@ -28,12 +28,12 @@
 
     private Pageable pageable;
 
-    public void setStartDate(LocalDate startDate) {
-        this.startDate = startDate.atStartOfDay(ZoneId.systemDefault());
+    public void setStartDate(LocalDateTime startDate) {
```

Comment:
why not set this at LocalDate so the controller passes it as LocalDate normally

Then this will just do atStartOfDay(ZoneId)

Reply:
dytafwgdjytfwajytdfdyt   K.